RunningOrder/RoundButton.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  RoundButton`.swift
+//  RunningOrder
+//
+//  Created by Lucas Barbero on 24/07/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import SwiftUI
+
+struct RoundButton: View {
+
+    let imageName: String
+    let color: Color
+    let action: () -> Void
+
+    var body: some View {
+        Button(action: self.action) {
+            Image(nsImage: NSImage(named: imageName)!)
```

Comment:
maybe pass the NSImage in parameter of the view instead of imageName, to pass whatever image we want instead

Reply:
or a SwiftUI Image directly